content/app/models/box.js

```diff
@@ -133,6 +158,36 @@ define("content/app/models/box", function(require, exports, module) {
 			return intersection;
 		},
 
+		/**
+		 * Distance to point. If point is in Box, then distance is 0.
+		 * @param  {Point} point
+		 * @return {Number}
+		 */
+		distanceToPoint : function (point) {
```

Comment:
Please add TODO if you are not going to retype it now, so that we do not forget.